src/components/Label/Label.scss

```diff
@@ -14,6 +14,20 @@ $transitionTimingFunction: ease-in-out;
         border: var(--border-size) solid #{$borderColor};
     }
 
+    // focus on interactive label (excluding focus on addon)
+    &:not(#{$disabled})#{$block}_is-interactive:focus:not(
+            :has(#{$block}__addon_interactive:focus)
```

Comment:
If Label is clickable and has close button this wouldn't allow to focus the Label itself